hack/charts/cluster-api-operator/values.yaml

```diff
@@ -1,12 +1,30 @@
 ---
 # ---
 # Cluster API provider options
-core: ""
-bootstrap: ""
-controlPlane: ""
-infrastructure: ""
-ipam: ""
-addon: ""
+core: {}
+  # name: ""
+  # namespace: ""  # Optional
+  # version: ""  # Optional
+bootstrap: []
```

Comment:
I personally like the fact a string and not a list since it's really hard with helm to override values in list.

If you move with change please consider a dictionary instead, something like:
```
bootstrap:
   <name>:
     namespace: "" # optional
     version: ""        # optional
     enabled: true   # optional
```

Reply:
That’s a valid argument and I’m not opposed to it. If this gets another vote from someone I’ll rework the PR.
Maybe @furkatgofurov7 can chime in?

Reply:
I am fine with it as well